test/fakeintake/client/client_test.go

```diff
@@ -497,4 +500,16 @@ func TestClient(t *testing.T) {
 			assert.Equal(t, expectedHostname, p.Hostname)
 		}
 	})
+
+	t.Run("getNDMFlows", func(t *testing.T) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can test the exported `GetNDMFlows` to ensure the external API is correct

Reply:
Good suggestion - done.